site/api/components/Class/Constructors.tsx

```diff
@@ -13,13 +15,15 @@ export function Constructors({
   if (!ctors.length) {
     return null;
   }
-  const items = ctors.map(({ accessibility, name, params }, i) => (
+  return ctors.map((v) => (
     <>
-      {accessibility ? `${accessibility} ` : undefined}
-      <span style="color: rgb(98, 232, 132);">{name}</span>(
-      <Params getLink={getLink}>{params}</Params>);
-      {"\n"}
+      <CodeBlock>
+        {v.accessibility ? `${v.accessibility} ` : undefined}
```

Comment:
v.accessibility should be inside StyleKw

Reply:
Right